hera_qm/xrfi.py

```diff
@@ -220,13 +220,22 @@ def xrfi(d, f=None, Kt=8, Kf=8, sig_init=6, sig_adj=2):
     return f
 
 
-def xrfi_run(files, args, history):
+def xrfi_run(files, args, history, broadcast=True, bl_threshold=0., freq_threshold=0.9,
+             time_threshold=0.9):
     """
     Run an RFI-flagging algorithm on an entire file and store results in flag array.
 
     Args:
        files -- a list of files to run RFI flagging on
        args -- parsed arguments via argparse.ArgumentParser.parse_args
+       broadcast -- (bool) If True (default), broadcast flags across data if
```

Comment:
Can we add the broadcast bool and threshold levels as options for the ArgumentParser instead? It's a little messy to mix command line arguments and keyword args.

Reply:
Whoops, I had that change sitting locally and forgot to commit. It's on its way.